MaskRCNN/eval.py

```diff
@@ -143,11 +144,11 @@ def predict_image_batch(img_batch, model_func, orig_sizes):
             boxes[ind, :] = boxes[ind, :]/scales[i]
 
             # boxes are already clipped inside the graph, but after the floating point scaling, this may not be true any more.
-            boxes[ind, :] = clip_boxes(boxes[ind, :], orig_sizes[i])
+            boxes[ind, :] = clip_boxes(boxes[ind, :], org_sizes[i])
```

Comment:
Any reason to `org_sizes` over `orig_sizes`? I think the latter is slightly clearer

Reply:
Fixed, `org_sizes` was legacy from an intermediate state of the code when both `orig_sizes` and `org_sizes` existed.